### PR TITLE
Feat/home page

### PR DESCRIPTION
**Pull request by SpicyI**
this PR:
- implements a home page 
- adds Eslint configs for better code quality
- sets the root layout

### CODE REVIEW

File: frontend/src/app/component/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+		if (searchValue) {
+			// Perform the search operation using the searchValue
+			console.log('Searching for:', searchValue);
+			// Add your search logic here
+		} else {
+			// Handle the case when searchValue is undefined
+			alert('Please enter a search term');
+			// Add your handling logic here
+		}
+	};
```
Comment (SpicyI) [inline frontend/src/app/component/NavBar.tsx:32]: this was generated by copilot

Comment (hmoumani) [reply frontend/src/app/component/NavBar.tsx:32]: are you that proud?

**Review by Hattaki-Hajar — APPROVED** [review]
LGTM

**Review by hmoumani — APPROVED** [review]
GG

File: frontend/src/app/component/NavBar.tsx (hunk 1 above)
Comment (SpicyI) [reply frontend/src/app/component/NavBar.tsx:32]: kinda

**Pull request merged**